packages/docusaurus-theme-classic/codeTranslations/he.json

```diff
@@ -0,0 +1,55 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "לסגור",
+  "theme.CodeBlock.copied": "הועתק",
+  "theme.CodeBlock.copy": "להעתיק",
+  "theme.CodeBlock.copyButtonAriaLabel": "להעתיק קוד",
+  "theme.NotFound.p1": "אנחנו לא מוצאים מה שאתה מנסה לחפש.",
+  "theme.NotFound.p2": "נא פנה למנהל האתר מאיפה קיבלת קישור הזה ותגיד לו שקישור לא תקין.",
+  "theme.NotFound.title": "דף לא נמצא",
+  "theme.Playground.liveEditor": "Live Editor",
+  "theme.Playground.result": "תוצאה",
+  "theme.PwaReloadPopup.closeButtonAriaLabel": "לסגור",
+  "theme.PwaReloadPopup.info": "קיימת גרסא חדשה",
+  "theme.PwaReloadPopup.refreshButtonText": "לרענן",
+  "theme.SearchBar.label": "חיפוש",
+  "theme.SearchPage.algoliaLabel": "חיפוש by Algolia",
+  "theme.SearchPage.documentsFound.plurals": "נמצא מסמך אחד|{count} מסמכים נמצאו",
+  "theme.SearchPage.emptyResultsTitle": "חפש בדוקומנטאציה",
+  "theme.SearchPage.existingResultsTitle": "תוצאות חיפוש עבור \"{query}\"",
+  "theme.SearchPage.fetchingNewResults": "מקבל תוצאות חדשות...",
+  "theme.SearchPage.inputLabel": "חיפוש",
+  "theme.SearchPage.inputPlaceholder": "הקלד כאן לחיפוש",
+  "theme.SearchPage.noResultsText": "לא נמצאו תוצאות",
+  "theme.blog.paginator.navAriaLabel": "רשימת דפים של הבלוג",
+  "theme.blog.paginator.newerEntries": "הכי חדש",
+  "theme.blog.paginator.olderEntries": "ישן יותר",
+  "theme.blog.post.paginator.navAriaLabel": "רשימת חדשות של הבלוג",
+  "theme.blog.post.paginator.newerPost": "הכי חדש",
+  "theme.blog.post.paginator.olderPost": "ישן יותר",
+  "theme.blog.post.plurals": "חדשה אחד|{count} חדשות",
```

Comment:
"he" has 4 plural forms, but this pluralized string only handle 2 plural forms

Reply:
@slorber This is actually incorrect.

One post:
רשומה אחת

{count} posts:
{count} רשומות

Reply:
That does make more sense

Reply:
🤪 thanks for the help.

My system need the 4 possible plural forms:

![image](https://user-images.githubusercontent.com/749374/118298981-7a274500-b4e0-11eb-94fd-434946d1e77b.png)

https://www.unicode.org/cldr/cldr-aux/charts/34/supplemental/language_plural_rules.html

If a plural form does not make sense, it's possible to use an empty string, but we should rather add the 4 plural forms in the correct order: one, two, many, other

Reply:
One:
רשומה אחת

Two:
שתי רשומות

Many, other:
{count} רשומות